kloppy/infra/serializers/event/wyscout/deserializer_v2.py

```diff
@@ -80,6 +80,62 @@ def _generic_qualifiers(raw_event: Dict) -> List[Qualifier]:
     return qualifiers
 
 
+def _create_shot_result_coordinates(raw_event: Dict) -> Optional[Point]:
+    """Estimate the shot end location from the Wyscout tags."""
+    if (
+        _has_tag(raw_event, wyscout_tags.GOAL_LOW_CENTER)
+        or _has_tag(raw_event, wyscout_tags.GOAL_CENTER)
+        or _has_tag(raw_event, wyscout_tags.GOAL_HIGH_CENTER)
+    ):
+        return Point(100.0, 50.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.GOAL_LOW_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_CENTER_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_HIGH_RIGHT)
+    ):
+        return Point(100.0, 55.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.GOAL_LOW_LEFT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_CENTER_LEFT)
+        or _has_tag(raw_event, wyscout_tags.GOAL_HIGH_LEFT)
+    ):
+        return Point(100.0, 45.0)
+    if _has_tag(raw_event, wyscout_tags.OUT_HIGH_CENTER) or _has_tag(
+        raw_event, wyscout_tags.POST_HIGH_CENTER
+    ):
+        return Point(100.0, 50.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.OUT_LOW_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.OUT_CENTER_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.OUT_HIGH_RIGHT)
+    ):
+        return Point(100.0, 60.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.OUT_LOW_LEFT)
+        or _has_tag(raw_event, wyscout_tags.OUT_CENTER_LEFT)
+        or _has_tag(raw_event, wyscout_tags.OUT_HIGH_LEFT)
+    ):
+        return Point(100.0, 40.0)
+    if (
+        _has_tag(raw_event, wyscout_tags.POST_LOW_LEFT)
+        or _has_tag(raw_event, wyscout_tags.POST_CENTER_LEFT)
+        or _has_tag(raw_event, wyscout_tags.POST_HIGH_LEFT)
+    ):
+        return Point(100.0, 55.38)
+    if (
+        _has_tag(raw_event, wyscout_tags.POST_LOW_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.POST_CENTER_RIGHT)
+        or _has_tag(raw_event, wyscout_tags.POST_HIGH_RIGHT)
+    ):
+        return Point(100.0, 44.62)
+    if _has_tag(raw_event, wyscout_tags.BLOCKED):
+        return Point(
```

Comment:
Do I understand correctly that the start location is also used as the end location when a shot is blocked because the actual location of the block can't be inferred from the data?

Reply:
Yes, I followed the implementation in socceraction. I guess it's the best possible estimate of the shot's end location. Another option would be to set it to `None`. Do you have a preference?

Reply:
Thanks! I think this approach is fine. I just wanted to check whether it was a deliberate choice or a potential mistake.